db/seeds.rb

```diff
@@ -240,7 +240,8 @@ def seed_quantity(item_name, organization, storage_location, quantity)
   distribution = Distribution.create!(storage_location: storage_location,
                                       partner: random_record_for_org(pdx_org, Partner),
                                       organization: pdx_org,
-                                      issued_at: (Date.today + rand(15).days))
+                                      issued_at: (Date.today + rand(15).days),
+                                      state: 0)
```

Comment:
It's fine to explicitly state this here, but since you did a `default` property for the DB, this isn't strictly necessary